lib/pcloud/files/file_handler.rb

```diff
@@ -1,4 +1,4 @@
-require 'RestClient'
+require 'rest_client'
```

Comment:
Can you rename it 'rest-client' ?

Basically 
- rest_client
- rest-client
- RestClient

All of these are okay, and i had been using `rest-client` in the `pcloud/client.rb`